resources/views/organizations/index.blade.php

```diff
@@ -0,0 +1,42 @@
+@extends('layouts.app')
+
+@section('content')
+    <div>
+        <div>
+            <h1>Organizations</h1>
+            @auth
+                <a href="{{ route('organizations.create') }}">Create new organization</a>
+            @endauth
+            @if ($organizations->count())
+                @foreach ($organizations as $organization)
+                    <div>
+                        {{ $organization->name }}
+                        {{ $organization->description }}
+                        {{ $organization->location }}
+                        {{ $organization->organization_type }}
+                        {{ $organization->foundation_date }}
+                        {{ $organization->number_of_employers }}
```

Comment:
```suggestion
                        {{ $organization->numberOfEmployers }}
```
If you use `CamelCaseAttributes` as I have indicated below you will be able to display the variables like this.